bdk_chain/tests/test_keychain_tracker.rs

```diff
@@ -74,7 +74,13 @@ fn test_balance() {
         input: vec![],
         output: vec![TxOut {
             value: 13_000,
-            script_pubkey: tracker.txout_index.derive_new(&Keychain::One).1.clone(),
+            script_pubkey: tracker
+                .txout_index
+                .derive_new(&Keychain::One)
+                .0
+                .unwrap()
+                .1
+                .clone(),
```

Comment:
Why not just put `DerivationAdditions` inside the `Option`? Much less friction that way and you don't care about it in the case where the option is `None`.